### DISCUSSION TITLE
How to format columns on the "data" tab?

### DISCUSSION THREAD

**Discussion by elriba** [pawelsalawa/sqlitestudio#4664] [Q&A]
Hi,
Is it possible to apply formatting to the columns on the "Data" tab for tables.   I refer to the "Grid view" of the data.

For example, I am looking at a table which defines columns "created_time" and "updated_time" as INT.   On the "data" tab they display as an integer value.  I am wondering if I can display them using a datetime format, or even a date format.

Also, there are NUMERIC columns which I would like to format with a given number of decimal points for example.

Is this possible?

Thanks!

**Comment by pawelsalawa** [comment]
That's what the SQL queries are for. Just select columns using whatever functions you need applied to certain columns.
